### PR TITLE
hikey960.xml: build using buildroot

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Change-Id: I65d6c20f4bd25464326ca8b4d330c9866b635015
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
Depends on https://github.com/OP-TEE/build/pull/250

**Comment by jforissier** [comment]
Merging https://github.com/OP-TEE/manifest/pull/109 has introduced a conflict.

**Comment by jforissier** [comment]
With the conflict fixed:
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`
`Tested-by: Jerome Forissier <jerome.forissier@linaro.org> (HiKey960)`

**Comment by jenswi-linaro** [comment]
Rebased and tags applied (and adjusted alignment of the variables in the buildroot line)

File: hikey960.xml [hunk 1]
```diff
@@ -14,18 +14,16 @@
...
         <project path="edk2"                  name="96boards-hikey/edk2.git"                  revision="2d899728eda95d57f8e9d9bee6cc9b6eefe422b7" />
         <project path="grub"                  name="grub.git"                                 revision="refs/tags/grub-2.02" clone-depth="1" remote="savannah" />
         <project path="linux"                 name="96boards-hikey/linux.git"                 revision="0e82bc3b3afaf5c154c781a5f62e1e9000e94e6a" />
         <project path="l-loader"              name="96boards-hikey/l-loader.git"              revision="c1cbbf8ab824820b5c1769a1c80dd234c5b57ffc" />
         <project path="OpenPlatformPkg"       name="96boards-hikey/OpenPlatformPkg.git"       revision="fbdd4aeee4d8de04d1c332379b20efb7a59a9502" />
-        <project path="strace"                name="strace/strace.git"                        revision="refs/tags/v4.21" clone-depth="1" />
         <project path="tools-images-hikey960" name="96boards-hikey/tools-images-hikey960.git" revision="a10d2bf1dca7a1be50fc60e58ed93253c95de076" />
+        <project path="buildroot"             name="buildroot/buildroot.git"                  revision="refs/tags/2017.11" clone-depth="1" />
```
Comment (jbech-linaro) [inline hikey960.xml]: I forgot about this on the ones merged yesterday, but --> https://github.com/OP-TEE/manifest#23-alphabetic-order

**Pull request merged**